examples/src/main/java/com/mapbox/navigation/examples/core/ReplayActivity.kt

```diff
@@ -57,18 +58,20 @@ class ReplayActivity : AppCompatActivity(), OnMapReadyCallback {
         mapView.onCreate(savedInstanceState)
 
         val mapboxNavigationOptions = MapboxNavigation.defaultNavigationOptions(
-            this,
-            Utils.getMapboxAccessToken(this)
+                this,
+                Utils.getMapboxAccessToken(this)
```

Comment:
Same as above re: unnecessary formatting changes

Reply:
made the formatting changes.